src/scripts/import-sdr-shipments-sheet/typeUtils.ts

```diff
@@ -0,0 +1,14 @@
+export const isDefined = <T>(v: T | undefined | null): v is T =>
+  v !== undefined && v !== null;
+
+export const isFulfilled = <T>(
+  value: PromiseSettledResult<T>,
+): value is PromiseFulfilledResult<T> => {
+  return value.status === "fulfilled";
+};
+
+export const _isRejected = <T>(
```

Comment:
**nitpick:** Should remove the _ in the function name.  That was originally included since we weren't using the function but it seemed useful to have around as a pairing with `isFulfilled`.  Now that it's refactored out and exported we don't need to indicate that it's not being used.